app/src/pages/inside/projectSettingsPageContainer/content/notifications/constants.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 EPAM Systems
```

Comment:
Please be more careful while copy/paste anything
update year everywhere to current one

Reply:
Yes, sorry, I saw it right after the commit